components/homme/src/theta-l_kokkos/theta_f2c_mod.F90

```diff
@@ -173,8 +173,9 @@ subroutine prim_run_subcycle_c(tstep,nstep,nm1,n0,np1,next_output_step,nsplit_it
     !
     ! Inputs
     !
-    integer(kind=c_int),  intent(in) :: nstep, nm1, n0, np1, next_output_step, nsplit_iter
-    real (kind=c_double), intent(in) :: tstep
+    integer(kind=c_int),  intent(inout) :: nstep
+    integer(kind=c_int),  intent(in)    :: nm1, n0, np1, next_output_step, nsplit_iter
```

Comment:
Note, first, that some of these others need out or inout, as well, to match the C++ rim_run_subcycle_c declaration. Second, preqx's F90 interface needs to be updated, too.

Reply:
thanks, will do.